docs/packaging_policy.md

```diff
@@ -0,0 +1,18 @@
+# Packaging Policy
+
+This document gathers all the relevant information regarding the general lines to follow while creating new recipes that will eventually be
+part of this repository.
+
+## Sources
+
+- Recipes should always build packages from library sources.
+
+- Library sources should come from an official origin like the library source code repository or the official release/download webpage.
+  Source forks or unofficial origin of sources are not allowed.
+
+- If library sources cannot be downloaded from their official origin or cannot be consumed directly due to their format, an issue should be
+  opened in order to discuss alternative solutions.
+
+- Library sources that are not publicly available will not be allowed in this repository even if the license allows their redistribution.
+
+- In the remote case of having to package precompiled libraries, those should come from official sources like release/download webpages.
```

Comment:
To add here (maybe reformulate together with the previous point): if binaries are not accessible the CTA is to contact the publisher and ask for a programmatically way to retrieve and consume those binaries.

If these binaries are needed as `build_requires` for some library, then we can write a `<app>/system` recipe and consider making those binaries available in the CI machines (if license let us install them) (#2850)